.idea/gradle.xml

```diff
@@ -19,4 +21,4 @@
       </GradleProjectSettings>
     </option>
   </component>
-</project>
+</project>
```

Comment:
We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

Reply:
To fix this You can also, just add blank line at the end.

Reply:
Can i add this file to git ignore it will happen for every PR